examples/LottieExtension.cpp

```diff
@@ -189,19 +219,25 @@ int main(int argc, char **argv)
 
         elm_init(argc, argv);
 
-        transit = elm_transit_add();
```

Comment:
Using multiple transitions here is inappropriate. Please refer to the examples/Lottie.cpp to see how it handles multiple animations.